revert.go

```diff
@@ -0,0 +1,103 @@
+package git
+
+/*
+#include <git2.h>
+*/
+import "C"
+import (
+	"runtime"
+)
+
+// RevertOptions contains options for performing a revert
+type RevertOptions struct {
+	Version      uint
```

Comment:
Version is not a user setting but decided at compile time to let libgit2 know what idea of the struct these bindings know about. It should always be set to `C.GIT_REVERT_OPTIONS_VERSION`. You use this for the default, but override for the case where someone passes in something. You should also prefer to use the function to initialize the default (though right now in this case it is admitedly not a complex default).

Reply:
Thanks for the feedback. `Version` has been removed from `RevertOptions`.  The value of `version` in the C struct is now always set to `C.GIT_REVERT_OPTIONS_VERSION`.